src/Storefront/Page/Robots/Parser/ParsedRobots.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page\Robots\Parser;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Storefront\Page\Robots\Struct\RobotsDirective;
+use Shopware\Storefront\Page\Robots\Struct\RobotsUserAgentBlock;
+
+#[Package('framework')]
+class ParsedRobots
+{
+    /**
+     * @param RobotsUserAgentBlock[] $userAgentBlocks
```

Comment:
```suggestion
     * @param list<RobotsUserAgentBlock> $userAgentBlocks
```

list is more precise. `string[]` can also be `array<string, string>` or `array<int, string>`

please change every occurrence